### PR TITLE
Fix the download-build command for Windows

### LINKED ISSUE #4901: sulu:admin:download-build fails on windows
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 2.0.2

#### Actual Behavior
```
In ResponseTrait.php line 235:
                                                                                                                              
  HTTP/1.1 400 Bad Request returned for "https://raw.githubusercontent.com/sulu/skeleton/2.0.2%5Cassets%5Cadmin%5Cindex.js".  
                                                                                                                              
```

#### Expected Behavior

Should download the admin build files.

#### Possible Solutions

The problem is that the DIRECTORY_SEPARATOR is being used to build urls in the getRemoteFileHash function and in windows the DIRECTORY_SEPARATOR  is "\\" not "/".
A solution would be:
```
$path = str_replace(DIRECTORY_SEPARATOR, "/", $path);
```
in that function.

### PR DESCRIPTION
**Pull request by lightglitch**
Fix #4901

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@lightglitch please updated your PR description with the issue template:

https://raw.githubusercontent.com/sulu/sulu/master/.github/ISSUE_TEMPLATE.md

File: src/Sulu/Bundle/AdminBundle/Command/DownloadBuildCommand.php [hunk 1]
```diff
@@ -154,6 +154,7 @@ private function getLocaleFileHash(string $path)
 
     private function getRemoteFileHash(string $path)
     {
+        $path = str_replace(DIRECTORY_SEPARATOR, "/", $path);
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/AdminBundle/Command/DownloadBuildCommand.php]: use single quotes here to fix pretty ci

**Comment by lightglitch** [comment]
Ok, I will do it a little later and fix the code cs.

**Comment by danrot** [comment]
This PR should be set against the `release/2.0` branch, otherwise it won't be published with Sulu 2.0.3... I am going to fix that for you, as well as the minor code style issue.

**Pull request merged**